util/downloadBuild.py

```diff
@@ -400,8 +403,9 @@ def defaultBuildType(repoName, arch, debug):
 
 def main():
     options = parseOptions()
+    if options.downloadFolder[-1] == '"': options.downloadFolder = options.downloadFolder[:-1];
```

Comment:
Sorry, I forgot I had added this line and therefore neglected to comment on it.

If you specify a path with spaces, you will need to use quotes. However, if you use <-w "my\download\folder\", it appears that the last quote is considered "escaped" by Windows and it will end up at the end of the option value.
This then causes the code to fail when it tries to install to <my\download\folder\"\build>. This extra line removes any trailing quote to fix this issue.